src/styles/signin_&_signup_stylesheet.css

```diff
@@ -20,12 +20,12 @@
 }
```

Comment:
The name of this file is not acceptable. Never do this again. Look at my comment about how you should organize your components and styles in @Verkijikavictorine 's PR

Reply:
Ok Sir thank you